plutus-ir/test/recursion/even3.plc

```diff
@@ -0,0 +1,135 @@
+(let
```

Comment:
This seems wrong - isn't this supposed to be using the definition with mutually recursive functions?

Reply:
(not super important since this particular test will probably go back to being synthesized)
(force-push overwrote this, same file can be found [here](https://github.com/m-alvarez/plutus/blob/pir-tests/plutus-ir/test/recursion/even3.plc))

It is, the first (nonrec) let binds the argument to the number three, and the second (recursive) let actually contains the mutual recursion (I didn't write the code by hand, this is just a copy-paste of `pretty evenOdd`)